pallene/driver.lua

```diff
@@ -133,14 +145,12 @@ local function compile_pln_to_lua(input_ext, output_ext, input_file_name, base_n
         return false, { err }
     end
 
-    local prog_ast, errs = driver.compile_internal(input_file_name, input, "checker")
-    if not prog_ast then
+    local translation, errs = driver.translate(input_file_name, input)
+    if not translation then
```

Comment:
I think we can keep all the logic inside compile_pln_to_lua like it was before. No need to split part of it into an `driver.translate` exported function.